_includes/registration-form.html

```diff
@@ -74,14 +79,14 @@
                 <legend>Ihre Kurse</legend>
                 <div id="basket">
                     <div id="gebuehrDiv" class="form-group row">
-                        <label for="gebuehr" class="col-form-label col-sm-4">Teilnahmegebühr</label>
+                        <label for="gebuehr" class="col-form-label col-sm-4" style="padding-top: 0.8em;">Teilnahmegebühr [€]</label>
```

Comment:
Ich würde direktes Styling per `@style` gerne vermeiden. Für zusätzliche Abstände kannst Du gut auch bootstrap-Klassen benutzen (https://getbootstrap.com/docs/4.0/utilities/spacing/) oder aber in `_sass/_ess.scss` unterbringen